PontoFacil/PontoFacil/Services/PersistencyService.cs

```diff
@@ -114,6 +115,23 @@ public Profile getProfile()
 
             return p != null ? p : new Profile();
         }
+
+        public List<ClockIn> GetMonthlyHistory()
+        {
+            DateTime firstDayOfTheMonth = new DateTime(DateTime.Now.Year, DateTime.Now.Month, 1);
```

Comment:
`new DateTime(DateTime.Now.Year, DateTime.Now.Month, 1)` extract the line the a global var named `FIRST_DAY_OF_MONTH`.